### DISCUSSION TITLE
Huge number of duplicate queries to admin User table

### DISCUSSION THREAD

**Discussion by matthewjumpsoffbuildings** [laravel/nova-issues#6292] [Q&A] [answered]
- Laravel Version: 10.48.3
- Nova Version: 4.33.0
- PHP Version: 8.3.0
- Database Driver & Version: MySQL 8

### Description:

I notice that within any given Nova request lifecycle, the same query

```sql
select * from `admins` where `id` = ? limit 1
```

Gets  run a _huge_ number of times, which appears totally unnecessary. The table `admins` is the table I have my Nova users on.

Surely within the confine of a single request lifecycle, once the Nova user has been accessed _once_ from the database, there is no need for subsequent authorization attempts to query the DB _again_? 

Once you have an authorized User model, why not hold it in memory and use it for the rest of the application request lifecycle?

**Comment by crynobone** [comment, marked as answer]
A page load in Laravel Nova will contain multiple small XHR requests to load the necessary information needed to populate the requested page, and each request will need to authenticate the current user. If that the case then this is to be the expected behavior.

**Reply by bohdan-shulha** [reply]
Would it be possible to batch the requests instead (on a framework-level)?
